### ISSUE TITLE
Withings konnector only imported the oldest 200 steps 

### ISSUE THREAD

**Issue by twitwi** [cozy-labs/konnectors#260]
I used the Withings connector and it imported most of my data (weight, heart rates, ...) but it did only import the first 200 logs of daily steps (the most recent ones are absent).

I tried re-runing the konnector, restarting the application and reconnecting. Both KYou and the data browser show only 200 steps. So my guess is that, maybe, the withings API provides chunks of 200 days.

**Comment by frankrousseau** [comment]
Yeah, I have a similar bug. It's related to API changes from Withings that occured earlier this year. I will check what could be done soon.

**Comment by frankrousseau** [comment]
Ok I made some research and I now know what to change. But it will require to rewrite properly the konnector. It will require some time to be done properly.

**Comment by twitwi** [comment]
Thanks for looking into it.
There is no urgency on my side. I can surely wait (or maybe even come back to it at some point and try to do the rewrite).
